src/engraving/rendering/dev/tlayout.cpp

```diff
@@ -4248,6 +4248,15 @@ void TLayout::layoutNote(const Note* item, Note::LayoutData* ldata)
         } else if (item->harmonic()) {
             const_cast<Note*>(item)->setHeadGroup(NoteHeadGroup::HEAD_DIAMOND);
         }
+
+        if (item->configuration()->guitarProImportExperimental()) {
```

Comment:
It would be better to add a new function to IEngravingConfiguration with a more explicit name and not to use guitarProImportExperimental() here. Ideally, we should get rid of IEngravingConfiguration::guitarProImportExperimental (it can still be used as a private function for the configuration, just remove it from the interface)

Reply:
thanks, I will add new method